libs/core/calendar/calendar-views/calendar-day-view/calendar-day-view.component.ts

```diff
@@ -845,6 +893,28 @@ export class CalendarDayViewComponent<D> implements OnInit, OnChanges, Focusable
         this.refreshTabIndex(calendar);
     }
 
+    /**
+     * @hidden
+     * Change selection flag on days to false, besides the selected one
+     *
+     * @param dates - An array of CalendarDay objects to be marked as selected
+     * @param calendar - The calendar array containing all CalendarDay objects
+     */
+    private _changeSelectedMultipleDays(dates: Array<CalendarDay<D>> | undefined, calendar: CalendarDay<D>[]): void {
```

Comment:
@dpavlenishvili 
I noticed that we already have a Nullable type to handle T | undefined cases. Would it be better to use that type here for consistency?